profiles/models.py

```diff
@@ -180,6 +179,7 @@ def import_customer_data(cls, data):
         And returns dict where key is the customer_id and value is the UUID of created profile object
         """
         result = {}
+        service = Service.objects.filter(name=service_name).first()
```

Comment:
I would prefer checking if the service_name is given and explicitly getting the one.

Reply:
Changed